File: support_tools/swigwin-3.0.12/Examples/test-suite/go/extend_template_runme.go
package main

import "./extend_template"

func main() {
	f := extend_template.NewFoo_0()
	if f.Test1(37) != 37 {
		panic(0)
	}

	if f.Test2(42) != 42 {
		panic(0)
	}
}

File: support_tools/swigwin-3.0.12/Examples/test-suite/ruby/extend_template_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# 
# 
# 
#

require 'swig_assert'

require 'extend_template'

f = Extend_template::Foo_0.new
if f.test1(37) != 37
  raise RuntimeError
end

if f.test2(42) != 42
  raise RuntimeError
end
